setup.cfg

```
@@ -134,9 +134,7 @@ install_requires =
     sqlalchemy>=1.4,<2.0
     sqlalchemy_jsonfield>=1.0
     tabulate>=0.7.5
-    # The 8.2.0 release of tenacity has a mypy error that breaks our CI
-    # The upper-bound limit can be removed after https://github.com/jd/tenacity/issues/389 is resolved
-    tenacity>=6.2.0,<8.2.0
+    tenacity>=8.2.1
```

```suggestion
    tenacity>=6.2.0,!=8.2.0
```

since the older versions aren’t incompatible?

---

Yeah. might be a good idea